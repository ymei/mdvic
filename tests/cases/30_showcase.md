# Showcase

This paragraph links to [OpenAI](https://openai.com) and has `inline code`.

> A quote with inline math and wrap: $a \leq b \implies a \leq b$.

- Item with inline math: $x^{2} + \sqrt[3]{y} = \alpha$
- Item with operators: $a \cdot b \subseteq C$, and composition $f \circ g$.

Header 1 | Center | Right
:--------|:------:|-----:
alpha    | beta   | gamma
$1/2$    | $x \rightarrow y$ | $A \iff B$

Greek variants: $\varepsilon, \varphi, \vartheta, \varrho, \varsigma$.

Accents: $\hat{x}, \bar{y}, \tilde{z}, \vec{v}, \ddot{q}, \dot{p}, \underline{r}$.

Arrows: $x \rightarrow y, \uparrow, \downarrow, \leftrightarrow, \implies, \iff$.

Delimiters: $\left\langle \frac{a}{b} \right\rangle$, $\left( \frac{1}{1+\frac{1}{n}} \right)$.

$$\sum_{k=0}^{m} \frac{\sin k}{k+1}$$

$$\prod_{j=1}^{3} j$$

$$\begin{pmatrix} 1 & 2 & 3 \\ a & b & c \\ \alpha & \beta & \gamma \end{pmatrix}$$

$$\begin{bmatrix} x & y \\ zz & w \end{bmatrix}$$

$$\begin{vmatrix} 1 & 2 \\ 3 & 4 \end{vmatrix}$$

```c
int main(void) {
    return 0;
}
```
