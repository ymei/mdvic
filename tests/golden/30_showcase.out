Showcase

This paragraph links to OpenAI (https://
openai.com) and has inline code.

> A quote with inline math and wrap: a ≤
>  b ⇒ a ≤ b.

- Item with inline math: x² + ³√(y) = α

- Item with operators: a · b ⊆ C, and co
  mposition f ∘ g.

| Header 1 |      Center       |      Right |
| :-------- | :-----------------: | ----------: |
| alpha    |       beta        |      gamma |
| 1/2    | x → y | A ⇔ B |


Greek variants: ϵ, ϕ, ϑ, ϱ, ς.

Accents: x̂, ȳ, z̃, v⃗, q̈, ṗ, r̲.

Arrows: x → y, ↑, ↓, ↔, ⇒, ⇔.

Delimiters: ⟨ a⁄b ⟩, ( 1⁄1+1⁄n ).

m
∑
k=0
sin k
-----
k+1


3
∏
j=1
j


(  1      2   3  )
(  a      b   c  )
(                )
( alpha          )
( beta           )
( gamma          )



[  x    y  ]
[  zz   w  ]



|  1   2  |
|  3   4  |



    int main(void) {
        return 0;
    }

